engine/runtime/src/test/java/org/enso/interpreter/test/BigNumberTest.java

```diff
@@ -97,6 +109,12 @@ private Value evalCode(final String code, final String methodName) throws URISyn
 
   @Test
   public void averageOfMixedArrayOverDouble() throws Exception {
+    boolean assertsOn = false;
+    assert assertsOn = true;
+    if (assertsOn) {
+      // skip this test when asserts are on
+      return;
```

Comment:
this warrants an extra ticket as well?

Reply:
There is https://www.pivotaltracker.com/story/show/183934416